### PR TITLE
Improve tests by adding check that receiver gets funds

### PR DESCRIPTION
**Pull request by lalexgap**

### CODE REVIEW

File: test/l1.test.ts [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
-let l1Contract: Contract;
 const ticketBatchSize = 10;
 const amountOfTickets = 100;
+const ticketValue = 10000;
+const depositValue = ticketValue * amountOfTickets * 100;
+
+type Balances = { sender: BigNumber; receiver: BigNumber };
+
+async function getBalances(): Promise<Balances> {
+  const receiverWallet = new Wallet(RECEIVER_PK, ethers.provider);
```
Comment (kerzhner) [inline test/l1.test.ts:22]: `receiverWallet` and `senderWallet` are constants defined at the top of the file. Any reason to redefine the constants in this function?

File: test/l1.test.ts [hunk 2]
```diff
@@ -21,32 +36,16 @@ describe("L1 Contract", function () {
...
-  });
-
   it(`can handle ${amountOfTickets} tickets being claimed sequentially`, async () => {
-    await l1Contract.deposit({ value: 10000 });
+    const initialBalances = await getBalances();
+    await l1Contract.deposit({ value: depositValue });
 
     const tickets: Ticket[] = [];
     const ticketSignatures = [];
     for (let i = 1; i <= amountOfTickets; i++) {
```
Comment (kerzhner) [inline test/l1.test.ts:46]: Not a part of this PR, but is there a reason that nonces start with 1 instead of 0?

**Review by kerzhner — APPROVED** [review]

File: test/l1.test.ts (hunk 2 above)
Comment (lalexgap) [reply test/l1.test.ts:46]: I originally wrote the contract checks to check that `ticket.nonce == senderNonces[ticket.sender] + 1` which meant that they had to start at `1`

That is addressed in https://github.com/statechannels/fast-exit/pull/20

File: test/l1.test.ts (hunk 1 above)
Comment (lalexgap) [reply test/l1.test.ts]: 2edee16

**Pull request merged**